chainercv/utils/iterator/apply_to_iterator.py

```diff
@@ -0,0 +1,167 @@
+from chainercv.utils.iterator.unzip import unzip
+
+
+def apply_to_iterator(func, iterator, n_input=1, hook=None):
+    """Apply a function/method to batches from an iterator.
+
+    This function applies a function/method to an iterator of batches.
```

Comment:
This function applies a function/method to batches from an iterator of type `chainer.Iterator`.

This makes clear that we are talking about Chainer's iterator.

Reply:
This is not only for `chainer.Iterator`. It can work with other iterators.  `an iterator of type chainer.Iterator` sounds it requires some special features.

Reply:
OK. Then we should change args doc.
https://github.com/Hakuyume/chainercv/blob/0bd2f6d970aeaca436e6858b0e53661e364c1fd6/chainercv/utils/iterator/apply_to_iterator.py#L75

Reply:
Thank you for pointing out.